### DISCUSSION TITLE
Queue button in the upper right disappeared

### DISCUSSION THREAD

**Discussion by GeorgiaM-honestly** [Comfy-Org/ComfyUI#5233] [Q&A]
Hello,

I haven't seen open bugs or anything about this so i'm assuming it's something local to my setups. I am seeing this happen on two different systems which are kept to being as close to identical as possible.

Starting on roughly 20241007, updates to comfyui via the manager resulted in the Queue button in the upper right (basically: start) being gone, which meant using comfyui was not possible. I keep backups and simply restored. However for me this has been going on almost a week now, since I just updated and am still experiencing the issue.

 - Systems: 2
 - OS: Ubuntu Server Linux
 - Running in a 3.12.3 virtualenv
 
 - System 1 specs:
Intel(R) Core(TM) i7-7820X CPU @ 3.60GHz
128GB RAM
Nvidia 3090 ti fi 24GB (Driver Version: 535.183.06)

 - System 2 specs:
Intel Xeon Silver (I forget which one i am not in front of that system at the moment)
296GB RAM
2 x Nvidia L40S 48GB (Driver Version: Unknown as i am not in front of that system at the moment)

I have tried using chrome, chromium, ungoogled-chromium, and firefox, all showing the same issue.

I always clear out all of my browser cache no matter what browser I am using almost daily, and I did again and restarted the mentioned browsers to no avail.

Please see the attached screenshot. Normally the Queue button is in the upper right, but instead all I see is "show image feed". I have poked around the UI trying to find out if the Queue button simply moved, however, I am not able to find it and I then revert to backups.

Thank you for reading!
![huh](https://github.com/user-attachments/assets/86dce1b1-519a-48fc-b452-2bd2d725e3a5)

**Comment by LukeG89** [comment]
It's not a bug, it's the new 1.3 frontend. You should have now the draggable/dockable Queue menu somewhere in your screen... But where is it??

![queue_menu](https://github.com/user-attachments/assets/5a65cb31-4698-46b0-b9e4-4c438a913cd2)


Try closing Image feed, maybe is under there

**Comment by ShunL12324** [comment]
have the same problem, cannot find the queue button
![image](https://github.com/user-attachments/assets/e7b9f4ce-653a-4bf9-a2a6-b4c1739afd75)

**Reply by LukeG89** [reply]
Clean all data of 127.0.0.1 site in your browser, this will reset the Queue menu position to the bottom of the screen.

**Comment by midareashi** [comment]
Zoom out (ctrl -) the browser tab a couple times and you should see it hiding in one of the corners. Move it to the center and zoom back to 100%.

**Reply by Arctomachine** [reply]
This is best answer. Without having to clear data for localhost, which might be used somewhere else.

**Reply by RobertDCBrown** [reply]
This was the fix for me, thank you!

**Reply by DenisRobocod** [reply]
Класс!!! Единственное решение.

**Comment by Infinitusvoid** [comment]
Well the auto queue is missing still !

**Reply by ltdrdata** [reply]
![image](https://github.com/user-attachments/assets/facda010-a843-4f6b-bbf7-31523b629bc8)

**Comment by ShunL12324** [comment]
Clean browser data works for me, then I put it into the top bar, Floating queue button could be a good idea but maybe we could have some thing like "reset layout" ?

Zoom out not work, it dissapeared completely

**Reply by LukeG89** [reply]
You can open a feature request in the frontend repo: [ComfyUI_frontend](https://github.com/Comfy-Org/ComfyUI_frontend)

**Comment by BrechtCorbeel** [comment]
Bros this looks better, but I just built a script for console for auto prompting and now it doesn't work anymore.

**Reply by BrechtCorbeel** [reply]
Ok, it still works, I think I can finally actually implement features in this GUI, but damn the tiling looks horrible. How am I supposed to navigate this?

![image](https://github.com/user-attachments/assets/e0fee540-c6a9-4bfc-b764-e95240646d46)

**Comment by charlierix** [comment]
Go to settings -> menu
Then disable the new style menu

I'm also not a fan of the new style trying to force me to use a buried folder for my workflow files, so double win

**Reply by Rushi-ai-web** [reply]
it worked

**Reply by Ntxtrung** [reply]
thanks, it was really worked

**Comment by rook2pawn** [comment]
![image](https://github.com/user-attachments/assets/a3ed1860-f641-48dd-976a-aa35f51a402e)
Lost the queue button myself. Zoomed out and its literally not there. This a bug or a feature?

**Reply by LukeG89** [reply]
You can try by zoomig out browser page, as suggested here: https://github.com/comfyanonymous/ComfyUI/discussions/5233?sort=new#discussioncomment-11338791

If that doesn't solve the issue, clear browser cache or just the stored data of 127.0.0.1 site. This will restore Queue menu position to the bottom-center of the screen.

**Comment by jakubGodula** [comment]
In my case, none of the solutions presented above worked.
Ctrl+F worked magically.

**Reply by vin901** [reply]
OMG thank you!
I was gonna say Ctrl+F is to find something on the page it must be a typo probably Ctrl+F5 to force refresh the page... 

But Ctrl+F worked!! 

That's so weird!!
